mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -248,12 +268,38 @@ class GBTClassificationModel private[ml](
     if (prediction > 0.0) 1.0 else 0.0
   }
 
+  override protected def predictRaw(features: Vector): Vector = {
+    val treePredictions = _trees.map(_.rootNode.predictImpl(features).prediction)
```

Comment:
we can avoid duplicating this code. Maybe, as in LogisticRegression, we can create a private function called `score` or `margin` and then use that in `predict` and `predictRaw`